tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-torch181-cuda111.yml

```diff
@@ -5,7 +5,7 @@ stages:
   parameters:
     build_py_parameters: --enable_training --update --build
     torch_version: '1.8.1'
-    cuda_version: '11.1'
+    cuda_version: '11.4'
```

Comment:
for training we need to publish wheels for different Cuda versions. As its name suggested, this pipeline is for Cuda 11.1. So please keep this file unchanged.